sqlalchemy-stubs/engine/base.pyi

```diff
@@ -36,9 +36,17 @@ class _ConnectionCallable(Protocol[_T_contra, _T_co]):
         self, __connection: _T_contra, *args: Any, **kwargs: Any
     ) -> _T_co: ...
 
-class Connection(Connectable):
-    engine: Engine = ...
+class _ConnectionProxy:
```

Comment:
why "Proxy" here?   proxy means a specific thing, that messages are being forwarded to the outside interface from another object that is maintained internally.   Maybe like "_ConnectionCommon" / "_EngineCommon", something like that since this is not even actually in the main code.

Reply:
Indeed proxy is not a good name for it

Reply:
I've suffixed them with TypingCommon instead